src/python/bot/tasks/task_creation.py

```diff
@@ -227,3 +237,57 @@ def create_tasks(testcase):
   # Just create the minimize task for now. Once minimization is complete, it
   # automatically created the rest of the needed tasks.
   create_minimize_task_if_needed(testcase)
+
+
+def _get_commits(commit_range, job_type):
+  """Get commits from range."""
+  start, end = revisions.get_start_and_end_revision(commit_range)
+  components = revisions.get_component_range_list(start, end, job_type)
+  if ':' not in components[0]['link_text']:
```

Comment:
assign components[0]['link_text'] to local and use in all places.

Reply:
done.